android-components/components/browser/state/src/test/java/mozilla/components/browser/state/engine/EngineObserverTest.kt

```diff
@@ -1307,26 +1307,6 @@ class EngineObserverTest {
         }
     }
 
-    @Test
```

Comment:
This test can be put back in until unified search is enabled by default on release.